cumulusci/robotframework/CumulusCI.py

```diff
@@ -241,3 +243,72 @@ def _run_task(self, task_class, task_config):
     def debug(self):
         """Pauses execution and enters the Python debugger."""
         set_pdb_trace()
+
+    def start_perf_timer(self):
+        """Start an elapsed time stopwatch for performance tests"""
+        BuiltIn().set_test_variable("${__start_time}", datetime.now())
+
+    def end_perf_timer(self):
+        """Record the results of a stopwatch. For perf testing.
+
+        This keyword uses Set Test Elapsed Time internally and therefore
+        outputs in all of the ways described there."""
+        builtins = BuiltIn()
+
+        start_time = builtins.get_variable_value("${__start_time}")
+        if start_time:
+            seconds = (datetime.now() - start_time).seconds
+            assert seconds is not None
+            self.set_test_elapsed_time(seconds)
+        else:
+            raise Exception(
+                "Elapsed time clock was not started. "
+                "Use the Start Elapsed Time keyword to do so."
+            )
+
+    def set_test_elapsed_time(self, elapsedtime):
+        """This keyword captures a computed rather than measured elapsed time for performance tests.
```

Comment:
I think it answers that question further down:

        Performance test times are output in the CCI logs and are captured in MetaCI instead of the
        "total elapsed time" measured by Robot Framework."""

Reply:
The last paragraph says the test times are output, but don't say where or how.

Reply:
Okay clarified:        

 Performance test times are output in the CCI logs and are captured in MetaCI instead of the
        "total elapsed time" measured by Robot Framework. The Robot "test message" is also updated."""

Reply:
That's in 0d1c8a2e